topologies/ate2dut.testbed

```diff
@@ -0,0 +1,39 @@
+# proto-file: github.com/openconfig/ondatra/blob/main/proto/testbed.proto
```

Comment:
Let's handle this addition in a separate PR. Some things you will need in that PR:

- First, can you rename this file atedutdut.testbed?
- You need to add an entry for the testbed here:
https://github.com/openconfig/featureprofiles/blob/main/proto/metadata.proto#L31
- A line here to map that enum value to the testbed file:
https://github.com/openconfig/featureprofiles/blob/main/internal/fptest/runtests.go#L70
- Under each of the kne topologies leaf directories, you'll need to add a matching KNE topology:
https://github.com/openconfig/featureprofiles/tree/main/topologies/kne/arista/ceos
- Then add a line here to map that testbed to that KNE topology:
https://github.com/openconfig/featureprofiles/blob/main/cloudbuild/virtual.sh
I think that's it, but I will double-check when you send me the PR

Reply:
Ack. Thanks.

Reply:
I have removed testbed file from this PR, will create new PR to address addition of new testbed. Thanks.

Reply:
@greg-dennis , I have created new PR for adding two DUT - ATE testbed file, please could you check.
https://github.com/openconfig/featureprofiles/pull/2105/files